src/main/webapp/detailsScript.js

```diff
@@ -0,0 +1,97 @@
+const API_KEY = 'AIzaSyA4o8AqxFRAFJD_S5-cEXPlJMqVeGHhGLI';
+// To get results from UK publishing houses.
+const country = 'UK';
+
+// Obtain the book ID from the quert string. This is the Google Books API ID.
+let string = window.location.search;
+let id = string.substring(string.lastIndexOf('=') + 1);
```

Comment:
You could change and use the `parseQueryString` function here for clarity, instead of using the fact that the ID is the last parameter.

Reply:
Will do